src/steps/03-Background/CurrentEnvironment/InstanceDetails.vue

```diff
@@ -525,6 +527,36 @@ export default class InstanceDetails extends Mixins(SaveOnLeave) {
     return isValid;
   }
 
+  /**
+   * Compiles and returns the sequence number for pricing details section
+   */
+  getPricingDetailsSequenceNum(): string {
```

Comment:
Please use getters for this and the method below. As getters in this use case, the code is only hit one time when DOM updates, but as methods, in some inputs, every time a change is made the method is hit again and again - for example, typing in the text input for additional information, the `getAdditionalInfoSequenceNum()` method below is hit with every keystroke. 
`public get getPricingDetailsSequenceNum(): string {` ... then in the markup remove the `()` so just `{{getPricingDetailsSequenceNum}}` in the markup